src/editor/app.js

```diff
@@ -390,6 +390,26 @@ UiDriver.registerEventHandler("C_CMD_SET_THEME", function(msg, data, prevReturn)
     editor.setOption("theme", data.name);
 });
 
+UiDriver.registerEventHandler("C_CMD_SET_FONT", function(msg, data, prevReturn) {
+
+    var fontSize = (data.size != "" && data.size>0) ? ("font-size:" + data.size + "px;") : "";
+
+    var fontFamily = data.family != "" ? ("font-family:'" + data.family + "';") : "";
+
+    var styleTag = document.getElementById('userFont');
+
+    if( styleTag  ){ 
+        styleTag.innerHTML = ".editor > .CodeMirror { " + fontFamily + fontSize + " }";
+        styleTag.appendTo('head');
```

Comment:
Even if a theme modifies the font on its own, the selector `.editor > .CodeMirror` is more specific (and can be made even more specific at will) so there is no problem, it always win.

If you can provide an example of a theme that modifies the font, then we can test if the current selector is specific enough.